blob/blob.go

```diff
@@ -172,7 +172,8 @@ func (a *Attributes) As(i interface{}) bool {
 type Writer struct {
 	b          driver.Bucket
 	w          driver.Writer
-	cancel     func() // cancels the ctx provided to NewTypedWriter if contentMD5 verification fails
+	end        func(error) // called to end the OpenCensus span in Close
```

Comment:
nit: use same comment as above, "called at Close to finish trace and metric collection"

Reply:
Done.